cli/command/image/trust.go

```diff
@@ -216,7 +216,8 @@ func imagePushPrivileged(ctx context.Context, cli *command.DockerCli, authConfig
 }
 
 // trustedPull handles content trust pulling of an image
-func trustedPull(ctx context.Context, cli *command.DockerCli, repoInfo *registry.RepositoryInfo, ref reference.Named, authConfig types.AuthConfig, requestPrivilege types.RequestPrivilegeFunc) error {
+func trustedPull(ctx context.Context, cli command.Cli, repoInfo *registry.RepositoryInfo, ref reference.Named,
+	authConfig types.AuthConfig, requestPrivilege types.RequestPrivilegeFunc) error {
```

Comment:
nit: I don't think we should do a return here (but really I'm ok merging it as is too 😛)